### ISSUE TITLE
Can I use my own container?

### ISSUE THREAD

**Issue by YangFei1990** [bytedance/byteps#74]
Hello. I was using Horovod for tensorflow distributed training for a long time. Given the performance comparison, I really want to try Byteps. However, I have a customized container which includes some dependencies and most importantly, our own version of tensorflow. Is it possible to run with our containers or do some modification with the official container?

**Comment by bobzhuyb** [comment]
Of course. You can build the byteps from code in your container. https://github.com/bytedance/byteps#build-from-source-code

The official container is just a convenient way to start, but certainly not the only way.
